docs/p12-limitation.rst

```diff
@@ -3,5 +3,21 @@
 Configure terminated HTTPS load balancer with PKCS12 certificate bundle
 =======================================================================
 
-Overview
---------
+F5 Neutron Lbaas Dashboard does not support load balancer SSL offloading function with PKCS 12 format certificates.
+
+The PKCS 12 bundle must be extracted before using F5 Neutron Lbaas Dashboard.
+
+- To extract a PKCS 12 bundle use openssl tool.
+
+.. code-block:: bash
+
+ # extract certificate with encrypt private keys
+ openssl pkcs12 -in certificate.pfx -out certificate.txt
+
+ # extract certificate without encrypt private key
+ openssl pkcs12 -in certificate.pfx -out certificate.txt -nodes
+
+- Make sure the extracted certificates are X509 format.
```

Comment:
'make sure' do you mean 'verify'?

Reply:
The user does not have to verify it to make sure the X509 format.

Reply:
I mean how does user to 'make sure'?